packages/aws-cdk/lib/init-templates/app/python/cdk.json

```diff
@@ -0,0 +1,3 @@
+{
+    "app": "python3 hello/hello_stack.py"
```

Comment:
Could this include running via the venv? Isn't that a better experience than requiring users to have pre-actived their shell and THEN running `cdk synth`?

Reply:
Yes, that's a good idea.  We should be able to just point to the python3 executable in the virtualenv.

Reply:
The EXE on Windows (for 3.7 at least as I just installed it last week) is just **python**.

Reply:
This gets ugly. If we are using a virtualenv that has been created using Python3 then there is no real need to say `python3` here.  However, that does mean that during the init process we have done a `python3 -m venv .env` so this, too, will fail on Windows.

We could try `python3` for the virtualenv and then try `python` if that fails but this would cause problems on a machine that does not have Python3 installed at all because we would end up with a virtualenv for Python2.

Maybe this is an argument against trying to create the virtualenv during the init process.

Reply:
It is possible to actually have the app entrypoint an executable file that takes care of all these details, so eventually `cdk.json` would be:

```json
{
  "app": "bin/my-awesome-app"
}
```